src/models/piranha_output.rs

```diff
@@ -16,21 +16,31 @@ use std::path::PathBuf;
 use itertools::Itertools;
 use serde_derive::Serialize;
 
+use crate::utilities::gen_py_str_methods;
+
 use super::{edit::Edit, matches::Match, source_code_unit::SourceCodeUnit};
-use pyo3::prelude::pyclass;
+use pyo3::{prelude::pyclass, pymethods};
+
+/// A class to represent Piranha's output
 #[derive(Serialize, Debug, Clone, Default)]
 #[pyclass]
 pub struct PiranhaOutputSummary {
+  /// path to the file
   #[pyo3(get)]
   path: String,
+  ///content of the file after all the rewrites
```

Comment:
```suggestion
  /// Content of the file after all the rewrites
```

Presumably, if this is the exported documentation?